java/ql/lib/semmle/code/java/dataflow/internal/DataFlowUtil.qll

```diff
@@ -304,6 +304,27 @@ class ContentSet instanceof Content {
   }
 }
 
+/**
+ * Holds if `g` validates the `e` upon evaluating to `branch`.
```

Comment:
nitpick, either of these would be fine for me -- but I would suspect the 1 or 3 to be the preferred one. I think I actually prefer the third one.

```suggestion
 * Holds if `g` validates `e` upon evaluating to `branch`.
```

```suggestion
 * Holds if `g` validates the expression `e` upon evaluating to `branch`.
```

```suggestion
 * Holds if the guard `g` validates the expression `e` upon evaluating to `branch`.
```